numpy/core/tests/test_umath.py

```diff
@@ -249,6 +249,29 @@ def test_division_int(self):
         assert_equal(x // 100, [0, 0, 0, 1, -1, -1, -1, -1, -2])
         assert_equal(x % 100, [5, 10, 90, 0, 95, 90, 10, 0, 80])
 
+    @pytest.mark.parametrize("input_dtype",
+            [np.int8, np.int16, np.int32, np.int64])
```

Comment:
Can you add timedelta64? assuming it is not super annoying...

Reply:
I have added some cases, not sure if those many cases are needed :). Please let me know if any changes are needed.

Reply:
Probably more than necessary, but that is fine. The different unit cases are not super interesting for the division code, but frankly a bit many tests are fine :).